src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementProjectService.java

```diff
@@ -366,35 +367,44 @@ public TeamMember addProjectTeamMember(final Integer projectId, final String use
-      // projectPackageSummary.setUri(getProjectUri);
-      projectPackageSummary.setAgreementId(mapping.getProject().getCaNumber());
-      projectPackageSummary.setLotId(mapping.getProject().getLotNumber());
-      projectPackageSummary.setProjectId(mapping.getProject().getId());
-      projectPackageSummary.setProjectName(mapping.getProject().getProjectName());
-
+    var agreementNo = mapping.getProject().getCaNumber();
+    var dbEvent = getCurrentEvent(mapping.getProject());
+    // TODO make single call instead of 2
+    try {
+      var agreementDetails = agreementsService.getAgreementDetails(agreementNo);
+      var lotDetails =
+          agreementsService.getLotDetails(agreementNo, mapping.getProject().getLotNumber());
+      projectPackageSummary.setAgreementName(agreementDetails.getName());
+      projectPackageSummary.setLotName(lotDetails.getName());
+    } catch (Exception e) {
+      //ignore for the moment, replace when single method to get all data from agreement service
+    }
+    // TODO no value for Uri
+    // projectPackageSummary.setUri(getProjectUri);
+    projectPackageSummary.setAgreementId(mapping.getProject().getCaNumber());
+    projectPackageSummary.setLotId(mapping.getProject().getLotNumber());
+    projectPackageSummary.setProjectId(mapping.getProject().getId());
+    projectPackageSummary.setProjectName(mapping.getProject().getProjectName());
+    //TODO use prodcument_events table to store status instead of call to Jaggaer
+    try {
       var exportRfxResponse = jaggaerService.getRfx(dbEvent.getExternalEventId());
-      var status = findTenderStatus(dbEvent,exportRfxResponse);
-      var eventSummary = tendersAPIModelUtils.buildEventSummary(dbEvent.getEventID(),
-          dbEvent.getEventName(), Optional.ofNullable(dbEvent.getExternalReferenceId()),
-          ViewEventType.fromValue(dbEvent.getEventType()), status, ReleaseTag.TENDER,
+      System.out.println("start "+LocalDateTime.now());
+      var status = findTenderStatus(dbEvent, exportRfxResponse);
+      System.out.println("End " + LocalDateTime.now());
+      var eventSummary =
+          tendersAPIModelUtils.buildEventSummary(dbEvent.getEventID(), dbEvent.getEventName(),
+              Optional.ofNullable(dbEvent.getExternalReferenceId()),
+              ViewEventType.fromValue(dbEvent.getEventType()), status, ReleaseTag.TENDER,
               Optional.ofNullable(dbEvent.getAssessmentId()));
-    eventSummary.tenderPeriod(new Period1()
-            .startDate(exportRfxResponse.getRfxSetting().getPublishDate())
-            .endDate(exportRfxResponse.getRfxSetting().getCloseDate()));
+      eventSummary.tenderPeriod(
+          new Period1().startDate(exportRfxResponse.getRfxSetting().getPublishDate())
+              .endDate(exportRfxResponse.getRfxSetting().getCloseDate()));
       projectPackageSummary.activeEvent(eventSummary);
-      return Optional.of(projectPackageSummary);
+    } catch (Exception e) {
+      // No data found in Jagger
```

Comment:
It would be worth at least logging this at `DEBUG` or `TRACE` level, perhaps with the value of  `dbEvent.getExternalEventId()` - might provide useful info in the future.

Reply:
Done

Reply:
I think the Slf4j  log statement placeholder is `{}` (not the `String` format placeholders such as `%s`, `%d` etc)